### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by anderson-dantas**

### CODE REVIEW

File: src/main/java/cloud/fogbow/rcs/api/http/response/Version.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package cloud.fogbow.rcs.api.http.response;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class Version {
+    @ApiModelProperty(example = "v.2.0.0-rcs-3191466-ras-ec6bf564-as-c803775-common-4e0d74e")
```
Comment (ANKerD) [inline src/main/java/cloud/fogbow/rcs/api/http/response/Version.java:6]: Please, change it to v1.

File: src/main/java/cloud/fogbow/rcs/constants/ConfigurationPropertyKeys.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package cloud.fogbow.rcs.constants;
+
+public class ConfigurationPropertyKeys {
+
+    public static final String BUILD_NUMBER_KEY = "build_number";
+    public static final String LOCAL_MEMBER_ID_KEY = "provider_id";
+    public static final String MEMBERSHIP_SERVICE_URL_KEY = "fogbow_membership_service_url";
```
Comment (ANKerD) [inline src/main/java/cloud/fogbow/rcs/constants/ConfigurationPropertyKeys.java:7]: Can you make this key simple? like 'ms_url'.

File: src/main/java/cloud/fogbow/rcs/core/ApplicationFacade.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import cloud.fogbow.common.util.ServiceAsymmetricKeysHolder;
+import cloud.fogbow.rcs.constants.ConfigurationPropertyDefaults;
+import cloud.fogbow.rcs.constants.ConfigurationPropertyKeys;
+import cloud.fogbow.rcs.constants.SystemConstants;
+import cloud.fogbow.rcs.core.models.ProviderMember;
+import cloud.fogbow.rcs.core.service.CatalogService;
+
+public class ApplicationFacade {
+    
+	private static final Logger LOGGER = Logger.getLogger(ApplicationFacade.class);
```
Comment (ANKerD) [inline src/main/java/cloud/fogbow/rcs/core/ApplicationFacade.java:20]: Fix this class identation please.

File: src/main/resources/application-dev.properties [hunk 4]
```diff
@@ -0,0 +1,9 @@
+server.port=8082
```
Comment (ANKerD) [inline src/main/resources/application-dev.properties:1]: I think this port is already taken by another Fog-service. Please change it to a higher number.

**Review by ANKerD — CHANGES_REQUESTED** [review]

**Review by pauloewerton — CHANGES_REQUESTED** [review]
@ANKerD can you please check whether this pr is actually valid?

**Pull request closed without merging**